test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -734,7 +734,7 @@ public void GetNuspec_ThrowsForNoRootNuspec()
                     var exception = Assert.Throws<PackagingException>(() => reader.GetNuspec());
 
                     // Assert
-                    Assert.Equal("Nuspec file does not exist in package.", exception.Message);
+                    Assert.Matches("^The global package folder is missing one or more files. Delete package folder and run the restore again: .*$", exception.Message);
```

Comment:
You should verify the error code.

Reply:
👍  I think the code is more important than the message, as it's something easier for customers to search for online. Plus  the fewer places that validate the message, the fewer tests to change when the message changes.